package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "openfin-service-async",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "check": "svc-tools check",
+    "clean": "rimraf dist",
+    "build": "npm run clean && tsc -p tsconfig.json && node src/fileCopy.js",
+    "deploy:local": "npm run build && cd dist && npm pack",
+    "fix": "svc-tools fix",
+    "test": "npm run test:unit",
+    "test:unit": "svc-tools test unit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/HadoukenIO/service-async"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/HadoukenIO/service-async/issues"
+  },
+  "homepage": "https://github.com/HadoukenIO/service-async",
+  "devDependencies": {
+    "@types/openfin": "^43.0.1",
+    "fs-extra": "^8.1.0",
+    "openfin-service-signal": "^1.0.0",
+    "openfin-service-tooling": "^1.0.22",
```

Comment:
You have this as a dev and peer dependency. Is that correct?

Reply:
Seems no!